ideasbox/templates/ideasbox/form_confirm_delete.html

```diff
@@ -0,0 +1,12 @@
+{% extends 'one-row.html' %}
+{% load i18n %}
+
+{% block row %}
+    <h2>{% block confirm_message %}{% endblock %}</h2>
+    <p>{% block confirm_warning %}{% endblock %}</p>
+    <form method="POST" action="." id="delete_form">
+        {% csrf_token %}
+        {{ form.as_p }}
+        <input type="submit" name="confirm" value="{% trans 'yes' %}" />
```

Comment:
This means all your fields will be inside a `<p>` elements, except for the submit button? For consistency, I tend to always put `<input>` element inside a `<p>` one.

But that's nitpicking. :smile: